### ISSUE TITLE
Tests fail in the brainvisa 5.0 bbi

### ISSUE THREAD

**Issue by ylep** [populse/capsul#174]
@brainvisa-release

```
FAILED: env PYTHONHASHSEED=0 /casa/host/build/bin/bv_env timeout -k 10 1500 python -m capsul.test

--------------------------------------------------------------------------------
----------------------------------------
'['/neurospin/brainvisa/build/Ubuntu-18.04-x86_64/bug_fix/bin/python', '/volatile/a-sac-ns-brainvisa/bbi-brainvisa-5.0/cea-5.0/src/development/casa-distro/5.0/bin/casa_distro', 'run', 'name=cea-5.0', 'env=BRAINVISA_TEST_RUN_DATA_DIR=/casa/host/tests/test,BRAINVISA_TEST_REF_DATA_DIR=/casa/host/tests/ref', '--', 'sh', '-c', 'env PYTHONHASHSEED=0 /casa/host/build/bin/bv_env timeout -k 10 1500 python -m capsul.test']'
----------------------------------------
......['/tmp/capsul_fslqLtDxL/bin/fsl5.0-bet', 'nothing', 'nothing_else']
..
================================================================================
Finished: FAILURE
```

See https://new.brainvisa.info/builds/job/cea-5.0/392/

**Comment by denisri** [comment]
Has the timeout been reached ? Difficult to know, the `timeout` command does not issue a message when it triggers, and we don't have job start / end / durations times in jenkins logs (this would be useful, together with the machine running tests).
Maybe we should write a variant of `timeout` which actually prints a comprehensive error when it reaches the timeout...

**Comment by denisri** [comment]
Actually there is a tooltip on jenkins jobs displaying their duration (but I don't see a place it's written clearly). However, yes, the job duration is 25 min, which is the default timeout, so we have to increase it.

**Comment by denisri** [comment]
Aha, there is no `CMakeLists` in pure python projects, thus no way to setup timeouts... We have to change the way we specify it in `info.py`...

**Comment by sapetnioc** [comment]
There is a `CMakeLists.txt` but it is harcoded, copied in build tree and is the same for all pure python projects.

**Comment by denisri** [comment]
`info.py` of projects (optinally) provide a tests list through the `test_commands` variable. The easiest now is to add another variable, `test_timeouts` which (optionally) provides timeouts for each test command both variables should be lists with the same length). It's not the best syntax, we would ideally switch to dict structures for test command items, but it would break compatibility.

**Comment by denisri** [comment]
On the other hand, taking 25 minutes for capsul test is not normal. Something must have hung somewhere in it.

**Comment by ylep** [comment]
@denisri I have restarted the tests a few minutes ago to see if they succeed now with the 1-hour timeout. Suspense on https://new.brainvisa.info/builds/...

**Comment by denisri** [comment]
I have seen that ;) and I have restarted tests for master too.
But now that I think about it I'm pretty sure increasing the timeout was not a good idea after all: we will just get failures later - when it goes right, capsul tests take less than 4 minutes.
There has been for a long time some random hangouts in soma-workflow (maybe at exit time) and I highly suspect we ran into that by chance last night.
